data/zilliqa/assets/zil1wh5z48kzn36jur7ffctcm09hl8e8jcypkpjzhe/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "GodZilliqa",
+    "symbol": "GODZ",
+    "supply": "2100",
+    "email": "GodZilliqaDeFi@gmail.com",
+    "product": true,
```

Comment:
remove this

Reply:
Man you are fast!

Reply:
I told you to just remove the product prop, not everything :)

Reply:
The whole thing was highlighted XD. Our telegram channel is part of our product, should I still remove it?